packages/x-date-pickers/src/internals/utils/valueManagers.ts

```diff
@@ -34,22 +34,29 @@ export const singleItemFieldValueManager: FieldValueManager<
 > = {
   updateReferenceValue: (utils, value, prevReferenceValue) =>
     value == null || !utils.isValid(value) ? prevReferenceValue : value,
-  getSectionsFromValue: (utils, localeText, prevSections, date, format) =>
-    addPositionPropertiesToSections(splitFormatIntoSections(utils, localeText, format, date)),
+  getSectionsFromValue: (utils, localeText, prevSections, date, format) => {
+    const shouldReUsePrevDateSections = !utils.isValid(date) && !!prevSections;
```

Comment:
Live for range fields, we now want the single item field to re-use the sections when the new value is invalid